### PR TITLE
Add blockchain address comments

### PR DESCRIPTION
**Pull request by IvanIvanoff**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/sanbase/blockchain_address/blockchain_address_comment.ex [hunk 1]
```diff
@@ -0,0 +1,27 @@
+defmodule Sanbase.BlockchainAddress.BlockchainAddressComment do
+  @moduledoc ~s"""
+  A mapping table connecting comments and short urls.
+
+  This module is used to create, update, delete and fetch insight comments.
```
Comment (tspenov) [inline lib/sanbase/blockchain_address/blockchain_address_comment.ex:5]: module description leftovers

File: lib/sanbase_web/graphql/dataloader/postgres_dataloader.ex [hunk 2]
```diff
@@ -91,6 +102,18 @@ defmodule SanbaseWeb.Graphql.MetricPostgresDataloader do
     |> Map.new()
   end
 
+  def query(:blockchain_addresses_comments_count, timeline_events_ids) do
```
Comment (tspenov) [inline lib/sanbase_web/graphql/dataloader/postgres_dataloader.ex:105]: leftover `timeline_events_ids`

File: priv/repo/structure.sql [hunk 3]
```diff
@@ -5388,3 +5465,5 @@ INSERT INTO public."schema_migrations" (version) VALUES (20201118125118);
 INSERT INTO public."schema_migrations" (version) VALUES (20201118141407);
 INSERT INTO public."schema_migrations" (version) VALUES (20201118145315);
 INSERT INTO public."schema_migrations" (version) VALUES (20201119085940);
+INSERT INTO public."schema_migrations" (version) VALUES (20201201102246);
```
Comment (tspenov) [inline priv/repo/structure.sql:5468]: Where is this one coming from ?

Comment (IvanIvanoff) [reply priv/repo/structure.sql:5468]: If you migrate without clearing the DB first, `ecto.dump` will add some stuff leftover in the DB from other branches. I'll remove it.

Comment (IvanIvanoff) [reply priv/repo/structure.sql:5468]: Actually the PR that adds this migration has not added the record here, so I'll leave it

File: test/sanbase_web/graphql/blockchain_address/blockchain_address_comment_api_test.exs [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+          insertedAt
+          editedAt
+      }
+    }
+    """
+
+    conn
+    |> post("/graphql", mutation_skeleton(mutation))
+    |> json_response(200)
+    |> get_in(["data", "updateComment"])
```
Comment (tspenov) [inline test/sanbase_web/graphql/blockchain_address/blockchain_address_comment_api_test.exs:166]: There are some shortcuts in helpers which replace these 4 lines with 1: 
for mutations: `execute_mutation(conn, mutation, "updateComment")`
for queries: `execute_query(conn, query, query_name)`

**Review by tspenov — APPROVED** [review]

**Pull request merged**